src/vs/workbench/browser/parts/editor/editorDropTarget.ts

```diff
@@ -21,11 +21,19 @@ import { IEditorService } from 'vs/workbench/services/editor/common/editorServic
 import { assertIsDefined, assertAllDefined } from 'vs/base/common/types';
 import { ITreeViewsService } from 'vs/workbench/services/views/browser/treeViewsService';
 import { isTemporaryWorkspace, IWorkspaceContextService } from 'vs/platform/workspace/common/workspace';
+import { IConfigurationService } from 'vs/platform/configuration/common/configuration';
 
 interface IDropOperation {
 	splitDirection?: GroupDirection;
 }
 
+function isDragIntoEditorEvent(configurationService: IConfigurationService, e: DragEvent): boolean {
+	if (!configurationService.getValue<boolean>('workbench.experimental.dragAndDropIntoEditor.enabled')) {
```

Comment:
Why do we have this setting actually? I would just enable this when `shiftKey` is held. A different setting should maybe be added for people that want the current drop behaviour only when `shiftKey` is hold.

Reply:
I'd like to keep this feature off by default while we are still figuring out the UX. We're still exploring what the correct keybinding to use here is and I don't want users to get used to using `shift` 

Once we are confident in which key we will use, I will explore adding a setting that lets users configure this